src/modules/ekf2/EKF/fake_pos_control.cpp

```diff
@@ -79,6 +79,9 @@ void Ekf::controlFakePosFusion()
 				}
 			}
 		}
+
+	} else if (_control_status.flags.fake_pos && isHorizontalAidingActive()) {
```

Comment:
Can't we simply wait (worst case 200ms) and let the fake pos fusion stop through `continuing_conditions_passing` being false?